yowsup/layers/protocol_media/protocolentities/message_media_downloadable.py

```diff
@@ -20,10 +20,10 @@ class DownloadableMediaMessageProtocolEntity(MediaMessageProtocolEntity):
     '''
     def __init__(self, mediaType,
             mimeType, fileHash, url, ip, size, fileName, 
-            _id = None, _from = None, to = None, notify = None, timestamp = None, participant = None,
-            preview = None, offline = None, retry = None):
+            _id = None, _from = None, to = None, notify = None, timestamp = None, 
+            participant = None, offline = None, retry = None):
 
-        super(DownloadableMediaMessageProtocolEntity, self).__init__(mediaType, _id, _from, to, notify, timestamp, participant, preview, offline, retry)
+        super(DownloadableMediaMessageProtocolEntity, self).__init__(mediaType, _id, _from, to, notify, timestamp, participant, offline, retry)
```

Comment:
The super constructor (MediaMessageProtocolEntity) is still expecting preview argument, so the preview, offline and retry got mixed up